### PR TITLE
Allow the app to run database migrations

### PR DESCRIPTION
**Pull request by tarebyte**
/cc @atareshawty

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 experiments/*
 node_modules/*
 bin/*
+db/*
```
Comment (atareshawty) [inline .eslintignore]: Instead of removing the directory from linting, can you fix the errors?

Comment (tarebyte) [inline .eslintignore]: Ugh that's a lot of errors...

Comment (atareshawty) [inline .eslintignore]: Are they auto-generated or copy paste?

**Comment by atareshawty** [comment]
So if I need to make a schema change, do migrations have to be written manually? Also, can you point me to some docs for all of this?

File: script/setup [hunk 2]
```diff
@@ -15,12 +15,25 @@ fi
...
+  echo "Development database already exists, dropping"
+  dropdb challenges_dev
 fi
-script/mock-data
 
+echo "===> Setting up development database..."
+createdb challenges_dev
+
+echo "==> Migrating development"
+./node_modules/db-migrate/bin/db-migrate up --config config/database.json -m db/migrations
```
Comment (btmills) [inline script/setup]: npm helpfully symlinks all bins (as long as they're declared as such in the module's `package.json`) to `./node_modules/.bin/bin_name`. Typically this path is preferred to manually referencing the path inside the module, as long as the module declares them.

**Comment by atareshawty** [comment]
@tarebyte  Thank you so much for this! I pulled a few ideas from it into #237 . I'm closing this, but it definitely helped me a lot. I appreciate it 🚀

**Pull request closed without merging**